server/src/app.ts

```diff
@@ -22,5 +22,6 @@ app.use(userRouter);
 app.use(feedRouter);
 app.use(projectRouter);
 app.use(milestoneRouter);
+app.use(projectRouter);
```

Comment:
you have projectRouter duplicated

Reply:
createProject will complain because it's expecting 2 arguments